src/redux/categories/categories.js

```diff
@@ -0,0 +1,22 @@
+// Actions
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Please follow the duck pattern for values of each action so it will be like:
```
const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```